.gitignore

```diff
@@ -31,3 +31,5 @@ yarn-error.log*
 # Editors
 .vscode/
 .idea/
+
+TODO
```

Comment:
is this needed?

Reply:
There is a VSCODE extension that you can use for adding **TODOs** so this was added intentionally to ignore `Todo`

Reply:
A comment then like  `# VSCode extensions` might be nice